### DISCUSSION TITLE
Meaning of github<

### DISCUSSION THREAD

**Discussion by vivekvardhan2810** [community/community#84971] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

means difference between git and github

**Comment by avaniishh123** [comment, marked as answer]
Git is a distributed version control system that tracks changes in source code, enabling collaboration among developers.
GitHub, on the other hand, is a web-based platform that provides hosting for Git repositories and offers additional collaboration features like issue tracking and pull requests.
Git is the underlying technology for version control, while GitHub is a service built on top of Git, providing a centralized platform for project management and collaboration.

**Comment by vivekvardhan2810** [comment]
thank you

**Comment by abhishehdhaladhuli** [comment]
Git is a popular version control system which is used by developers to track changes in the code and it is considered to be fast, open source and scalable .On the other hand Github is a web platform which is mainly used to host the repositories and it can be treated like codes garage
